# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/cubeb-coreaudio/run_device_tests.sh
# Commit: ad954710c53f
# Full Hash: ad954710c53fa84879169b4f3d7994bb5f2ab714
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2024-06-03 16:07:28
# Regressor Bug: 1890186
# File Overlap Count: 5
# Description:
#   Bug 1890186: Update cubeb-coreaudio-rs to 1796ace5bd. r=cubeb-reviewers,padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D212311
# ==============================================================================

diff -r 8952a815f4b4 -r ad954710c53f third_party/rust/cubeb-coreaudio/run_device_tests.sh
--- a/third_party/rust/cubeb-coreaudio/run_device_tests.sh	Mon Jun 03 13:31:38 2024 +0300
+++ b/third_party/rust/cubeb-coreaudio/run_device_tests.sh	Mon Jun 03 10:03:39 2024 +0000
@@ -8,10 +8,15 @@
 fi
 echo "RUST_BACKTRACE is set to ${RUST_BACKTRACE}\n"
 
+cargo test test_aggregate -- --ignored --nocapture
+
 cargo test test_switch_device -- --ignored --nocapture
 
 cargo test test_plug_and_unplug_device -- --ignored --nocapture
 
+cargo test test_get_channel_count_of_input_devices_with_vpio -- --ignored --nocapture
+cargo test test_get_channel_count_of_input_devices_with_aggregate_device_and_vpio -- --ignored --nocapture
+
 cargo test test_register_device_changed_callback_to_check_default_device_changed_input -- --ignored --nocapture
 cargo test test_register_device_changed_callback_to_check_default_device_changed_output -- --ignored --nocapture
 cargo test test_register_device_changed_callback_to_check_default_device_changed_duplex -- --ignored --nocapture